pkg/builder/cluster/builder.go

```diff
@@ -21,16 +21,16 @@ import (
 	"fmt"
 	"sync"
 
+	v1alpha1 "github.com/knative/build/pkg/apis/build/v1alpha1"
```

Comment:
Can we separate these first-party imports (anything from `knative/build`) from the vendored imports from other packages?